repositories/decisions_repository.go

```diff
@@ -71,6 +72,54 @@ func (repo *MarbleDbRepository) DecisionWithRuleExecutionsById(ctx context.Conte
 	return decisions[0], nil
 }
 
+func (repo *MarbleDbRepository) DecisionsByOutcome(ctx context.Context, exec Executor,
+	scenarioID string, begin, end time.Time,
+) ([]models.DecisionsByVersionByOutcoume, error) {
+	query := NewQueryBuilder().
+		Select("d.outcome, d.scenario_version, d.score, Count(d.outcome) as total").
+		From(fmt.Sprintf("%s AS d", dbmodels.TABLE_DECISIONS)).Where(squirrel.Eq{
+		"d.scenario_id": scenarioID,
+	}).Where(squirrel.Eq{
+		"d.created_at": begin.String(),
+	}).Where(squirrel.Eq{
+		"d.deleted_at": end.String(),
+	}).GroupBy("d.outcome, d.scenario_version, d.score")
+	return SqlToListOfRow(ctx,
+		exec,
+		query,
+		func(row pgx.CollectableRow) (models.DecisionsByVersionByOutcoume, error) {
+			db, err := pgx.RowToStructByPos[dbmodels.DbDecisionsByOutcome](row)
+			if err != nil {
+				return models.DecisionsByVersionByOutcoume{}, err
+			}
+			return dbmodels.AdaptDecisionByOutcome(db), nil
+		})
+}
+
+func (repo *MarbleDbRepository) DecisionsByScore(ctx context.Context, exec Executor,
```

Comment:
As far as I understand, those two sql queries return exactly the same results (in fact, the only difference is `count(d.score)` vs `count(d.outcome)`, which is equivalent here seeing as it counts rows)
Either way, do we need the two ?
(if we do want a version that groups only by outcome, which is to be discussed with Thomas and Kevin tomorrow, it should have a less granular GROUP BY condition)